src/profiler.h

```diff
@@ -47,6 +47,24 @@ enum State {
     TERMINATED
 };
 
+enum EventMask {
+    EM_CPU          = 1,
+    EM_ALLOC        = 2,
+    EM_LOCK         = 4,
+    EM_WALL         = 8,
+    EM_NATIVEMEM    = 16,
+    EM_METHOD_TRACE = 32,
+};
+constexpr u8 EVENT_MASK_SIZE = 6;
+
+static int make_event_mask(const Arguments& args) {
+    return (args._event     != NULL ? (args._latency >= 0 ? EM_METHOD_TRACE : EM_CPU) : 0) |
+           (args._alloc     >= 0    ? EM_ALLOC        : 0) |
```

Comment:
nit: If we are doing the `?` alignment, can we align the last `:` characters as well?

Reply:
3931b663afad75a7b9a03ca40ad4e80e56547eb3